boa_engine/src/value/mod.rs

```diff
@@ -1000,6 +1000,13 @@ impl From<f32> for Numeric {
     }
 }
 
+impl From<i128> for Numeric {
+    #[inline]
+    fn from(value: i128) -> Self {
+        Self::BigInt(value.into())
+    }
+}
```

Comment:
We should remove this, as well as `From<u128> for Numeric`. If the user wants to a bigint or number they should explicitly say so.

Besides it's out of the scope of this PR of adding `From<i/u128> for JsBigInt`